HomeFinder/Models/ListAttributesRepository.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HomeFinder.Models
+{
+    public class ListAttributesRepository //: IAttributesRepository
+    {
+        //this class was the first implementation of CRUD for LocationAttributes
+        //I kept it because it's work that I have done and don't want to be lost
+        //Because of Repository pattern it doesn't interfere with the functionality of the program
+        //as it is no longer being used
+        //It doesn't take fully in consideration the changes in Rank of the attributes
+        List<LocationAttributes> _attributesList;
+        public ListAttributesRepository()
+        {
+            _attributesList = new List<LocationAttributes>
+           {
+               new LocationAttributes{Id = 1, Name = "Aproape de parc", Rank = 1},
```

Comment:
don't hardcode values if you don't need them

Reply:
This was just for learning. In the database class, I didn't hardcode anything.